api_v3/services/UserService.php

```diff
@@ -76,17 +76,32 @@ public function updateAction($userId, KalturaUser $user)
 			throw new KalturaAPIException(KalturaErrors::INVALID_USER_ID, $userId);
 		}
 
+		$partner = $dbUser->getPartner();
 		if ($dbUser->getIsAdmin() && !is_null($user->isAdmin) && !$user->isAdmin)
 		{
 			throw new KalturaAPIException(KalturaErrors::CANNOT_SET_ROOT_ADMIN_AS_NO_ADMIN);
 		}
+
+		if ($dbUser->getIsAdmin() && !is_null($user->email))
```

Comment:
write the order of lines the same as in the way you implement above in alpha/lib/model/kuserPeer.php